gradle/tools/detekt/detekt-config.yml

```diff
@@ -119,7 +119,7 @@ complexity:
   LongParameterList:
     active: true
     functionThreshold: 6
-    constructorThreshold: 7
+    constructorThreshold: 8
```

Comment:
bumped it up because I couldn't think of any other way to solve this in AnalyticsTracker now that it has 7 dependencies. see what you think about it.

Reply:
AnalyticsTracker could perhaps remove the `AppcuesRepository` dependency? And then inject that (and coroutine scope) into the new `AnalyticsQueueManager` - since that is the spot where it actually makes the network call now.  Then you also would not need to pass the `send` closure along.  Would that simplify?